K2Bridge/Visitors/QueriesVisitor.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+namespace K2Bridge.Visitors
+{
+    using K2Bridge.Models.Request.Queries;
```

Comment:
Why have the visitor and entity been to renamed to **Queries**?

Reply:
A remnant of the old proposal, will revert